accelerator-core/src/main/java/com/tokbox/android/otsdkwrapper/listeners/UnfailingAdvancedListener.java

```diff
@@ -54,6 +54,32 @@ public void onReconnected(Wrapper wrapper) {
         }
     }
 
+    @Override
+    public void onReconnected(Wrapper wrapper, String remoteId) {
+        if (mInternalListener != null) {
+            mInternalListener.onReconnected(wrapper, remoteId);
+        }
+    }
+
+    @Override
+    public void onDisconnected(Wrapper wrapper, String remoteId) {
+        if (mInternalListener != null) {
+            mInternalListener.onDisconnected(wrapper, remoteId);
+        }
+    }
+
+    @Override
+    public void onAudioEnabled(Wrapper wrapper, String remoteId) {
+        if (mInternalListener != null) {
+            mInternalListener.onAudioEnabled(wrapper, remoteId);
+        }
+    }
+
+    @Override
+    public void onAudioDisabled(Wrapper wrapper, String remoteId) {
```

Comment:
Could be a dumb question. Should mInternalListener. onAudioDisabled be invoked here?

Reply:
good catch 👍